Remove extra "like" from sentence (#2053)
graphql/graphql.github.io: src/pages/learn/mutations.mdx

Mutation fields can also accept arguments and you might notice that the `review` argument has an input type set to `ReviewInput`. This is known as [Input Object type](/learn/schema/#input-object-types), which allows us to pass in a structured object containing information the mutation can use instead of individual scalar values only.

> Also just like queries, if the mutation field returns an Object type, then you specify a selection set of its fields in the operation:

Reviewer: ```suggestion
As with queries, if the mutation field returns an Object type then you must specify a selection set of its fields in the operation:
```
